sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -406,6 +406,36 @@ ze_result_t ZeCall::doCall(ze_result_t ZeResult, const char *CallStr,
     return mapError(Result);
 #define ZE_CALL_NOCHECK(Call) ZeCall().doCall(Call, #Call, false)
 
+inline void piQueueRetainNoLock(pi_queue Queue) { Queue->RefCount++; }
+
+// This helper function creates a pi_event and associate a pi_queue.
+// Note that the caller of this function must have acquired lock on the Queue
+// that is passed in.
+// \param Queue pi_queue to associate with a new event.
+// \param Event a pointer to hold the newly created pi_event
+// \param CommandType various command type determined by the caller
+// \param ZeCommandList the handle to associate with the newly created event
+// \param ZeEvent the ZeEvent handle to be associated with the event
+inline pi_result createEventAndAssociateQueue(
```

Comment:
does the "inline" make it "static" too? if not please add static.
actually you can remove inline in this case

Reply:
"inline" is typically used when outlining a common code section as a separate function so that the compiler can actually put it back into the place (inline).
It does not mean "static" scope specifier.
Are you okay with changing it to "inline static" instead of only "static"?

Reply:
Done. I added "static" in addition to "inline". Let me know if you still want to remove "inline".